templates/windows-admin-center-extension-template/gulpfile.ts/common/utilities.ts

```diff
@@ -34,9 +35,10 @@ module Utilities {
     }
 
     export function ng(cb, args, options = {}, codeHandler = null) {
-        log('ng', args.join(' '));
+        const ng_cmd = appRoot.path + '\\node_modules\\.bin\\ng.cmd';
+        log(ng_cmd, args.join(' '));
         const errors = [];
-        const cmd = childProcess.spawn('ng.cmd', args, options);
+        const cmd = childProcess.spawn(ng_cmd, args, options);
```

Comment:
It looks like this can be done by just using "childProcess.spawn('node_modules\\.bin\\ng.cmd', args, options);" without using an additional package.

Reply:
Sometimes Paths can be tricky and I wanted to be safe. If it makes you feel better, the package is already in the package-lock as another package depends on it.

That said I have no objection simplifying it if we are confident there's no case where not being explicit with the path would cause harm.

Reply:
Oh got it, it's good to leave then if we're already taking the package anyway through a dependency.

Reply:
There may be a better way to do this entirely, but I wasn't sure how to do it in code. On cmdline one can run `npm bin` to get the bin folder, but I was hesitant to spawn another process to get that path, and couldn't identify how to access that api from within the JS.